galley/pkg/server/configmap.go

```diff
@@ -61,15 +91,17 @@ func watchAccessList(stopCh <-chan struct{}, accesslistfile string) (*server.Lis
 
 		for {
 			select {
-			case e := <-watcher.Event:
-				if e.IsCreate() || e.IsModify() {
+			case e := <-watcher.Events():
+				if e.Op&fsnotify.Write == fsnotify.Write {
 					if list, err = readAccessList(accesslistfile); err != nil {
 						log.Errorf("Error reading access list %q: %v", accesslistfile, err)
```

Comment:
nit: while you are in here, does it make sense to change this to a camel-case to match the rest of the file?